posix-configs/SITL/init/rcS_jmavsim_iris

```diff
@@ -53,7 +53,8 @@ sensors start
 commander start
 land_detector start multicopter
 navigator start
-ekf2 start
+attitude_estimator_q start
+local_position_estimator start
```

Comment:
This is a change that we need to debate, which brings us to the ultimate EKF2 vs attitude_estimator_q + LPE question.